### PR TITLE
[zh] Sync releasenote 1.19.6 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync releasenote 1.19.6 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Review by hanxiaop — COMMENTED** [review]
Can we split this into two separate PRs? 1.19.6 has been released and can be cherry-picked. However 1.20.2 is still in progress.

**Comment by wilsonwu** [comment]
> Can we split this into two separate PRs? 1.19.6 has been released and can be cherry-picked. However 1.20.2 is still in progress.

OK, no problem.

File: content/zh/news/releases/1.19.x/announcing-1.19.6/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ([Issue #48336](https://github.com/istio/istio/issues/48336))
+
+- **修复** 修复了 `istioctl proxy-config ecds` 未显示所有 `EcdsConfigDump` 的问题。
+
+- **修复** 修复了新端点可能无法被发送到代理的问题。
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **修复** 修复了使用 Stackdriver 安装并使用自定义配置会阻止 Stackdriver 被启用的问题。
+
+- **修复** 修复了 TCP 和 gRPC 长连接导致代理内存泄漏的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19.6/index.md:33]:
```suggestion
- **修复** 修复了 TCP 和 gRPC 长连接可能导致代理内存泄漏的问题。
```

**Review by windsonsea — COMMENTED** [review]
Pick a nit

**Review by windsonsea — APPROVED** [review]
thanks, lgtm

**Comment by windsonsea** [comment]
hmmm, is it correct to cherry pick to 1.19? @hanxiaop

**Comment by windsonsea** [comment]
Thanks, got it.

**Pull request merged**